src/mscorlib/src/System/Resources/ResourceManager.cs

```diff
@@ -99,9 +98,8 @@ internal class PRIExceptionInfo
     // RM will need to use file-based or manifest-based resources, and 
     // afterwards refers to this through the interface IResourceGroveler.
     // 
-    // Serialization Compat: Ideally, we could have refactored further but
-    // this would have broken serialization compat. For example, the
-    // ResourceManager member UseManifest and UseSatelliteAssem are no 
+    // TODO: more serialization cleanup as type isn't serializable anymore.
```

Comment:
Good point, not yet. Let me first discuss that matter with the area owner.

@tarekgh the issue guide says you are area owner of System.Resources. The comments in ResourceManager suggest that code refactoring could/should be done when serialization support is stripped from the type. This is now the case. Is the comment still accurate, are there optimizations possible now?